mantidproject/mantid
# CreateLogPropertyTable improvements

Fixes #12796.

This improves handling of time series properties and allows the results to be easily plotted.

To test:
- The OSIRIS temperature scan in `babylon5/Public/DanNixon/OSIRIS_scan`
- Open the CreateLogPropertyTable algorithm
- Set `InputWorkspaces` to the name of the workspace group
- Set `LogpropertyNames` to "run_number,stick,h2_moderator,Main_Vessel_Pressure"
- Run
- Plot `stick` vs `run number` (should be possible by right clicking on stick heading and selecting Plot) (`stick` is the temperture of the sample stick)

Release notes changes [here](http://www.mantidproject.org/index.php?title=ReleaseNotes_3_5_Framework_Changes&diff=24278&oldid=24277).
